apps/vetrina/src/Vetrina/Main.purs

```diff
@@ -273,20 +277,42 @@ orderErrorMessage :: OrderFailure -> JSX
 orderErrorMessage failure =
   case failure of
     AuthenticationError -> InputField.errorMessage "Kombinationen av e-postadress och lösenord finns inte"
-    EmailInUse -> DOM.text "Email already exists, please log in" -- TODO: Waiting for copy
+    EmailInUse -> mempty -- TODO: Waiting for copy
     _ -> DOM.text "Något gick fel. Vänligen försök om en stund igen."
 
-newAccountForm :: Self -> Array JSX -> JSX
-newAccountForm self children =
+title :: Self -> JSX
+title self@{ state: { accountStatus } } = case accountStatus of
+                                            NewAccount ->      DOM.text "Hej kära läsare!"
+                                            ExistingAccount -> DOM.text "Du har redan ett KSF Media-konto"
+
+description :: Self -> JSX
+description self@{ state: { accountStatus } } = case accountStatus of
+                                                  NewAccount ->      DOM.text "Den här artikeln är exklusiv för våra prenumeranter."
+                                                  ExistingAccount -> DOM.text "Vänligen logga in med ditt KSF Media lösenord."
+
+resetPasswordLink :: JSX
+resetPasswordLink = DOM.p_
+                      [ DOM.text "Glömt lösenordet? "
+                      , DOM.a
+                          { href: "https://www.hbl.fi/losenord/"
+                          , children: [ DOM.text "Klicka här"]
```

Comment:
We need styles for the link (remove underline, add proper color). I guess the color should be `#00a1ab` if we follow the colors from sites (look e.g. https://www.hbl.fi/artikel/mycket-sprit-i-karantanhemmen-kan-valla-problem-pa-lang-sikt-2/)

Reply:
Style added